lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -5030,47 +5045,81 @@ int main(int argc, char *argv[]) {
   }
 #endif
 
+  std::unique_ptr<std::ofstream> log = nullptr;
+  const char *log_file_path = getenv("LLDBDAP_LOG");
+  if (log_file_path)
+    log.reset(new std::ofstream(log_file_path));
```

Comment:
```suggestion
    log = std::make_unique<std::ofstream>(log_file_path));
```

to avoid the bare `new`.

Reply:
Done.